tests/integration/tests.py

```diff
@@ -10,6 +10,9 @@
 import test_client
 
 djl_version = os.environ.get('TEST_DJL_VERSION', '').strip()
+override_image_tag_suffix = os.environ.get('OVERRIDE_IMAGE_TAG_SUFFIX',
+                                           '').strip()
+image_repo = os.environ.get('IMAGE_REPO', '').strip()
```

Comment:
What's the reasoning for two separate env vars here? Would it be simpler to just have a single `OVERRIDE_CONTAINER_URI` env var?

Reply:
since we have multiple arches, the tags will be arch-{suffix}. Having  `OVERRIDE_CONTAINER_URI` here will require us to generate image uri for each arch.